docs/content/about-iota/iota-wallet/how-to/integrate-ledger.mdx

```diff
@@ -40,7 +40,7 @@ Make sure you have enough space left for an additional app on your device.
 
 :::
 
-1. Go to the latest release on https://github.com/iotaledger/ledger-app-iota/releases.
+1. Go to the latest release on https://github.com/iotaledger/ledger-app-iota/releases/tag/ledger-app-iota-v0.9.2.
```

Comment:
Do you not think we should keep this page generic for any release?

Reply:
No, tbh I don't expect us to do another release for the ledger nano s any time soon and the identifier I added below is only valid for this release
If we do another release, I would then just adjust the docs again

Reply:
Fair enough